dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -753,9 +753,10 @@ public void handleInfo(String entryPath) {
                 errorMessage("This container is not published.", Client.COMMAND_ERROR);
             } else {
 
-                Date lastBuild = container.getLastBuild();
+                final Long lastBuildLong = container.getLastBuild();
                 Date dateUploaded = null;
-                if (lastBuild != null) {
+                if (lastBuildLong != null) {
+                    final Date lastBuild = new Date(lastBuildLong);
```

Comment:
You could just do `dateUploaded = new Date(lastBuildLong)`, instead of creating an intermediate Date, but this is following the existing pattern, and maybe there's subtlety that I'm missing.